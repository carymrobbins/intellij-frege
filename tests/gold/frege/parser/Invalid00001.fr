module Invalid wher

cats = "dogs"
