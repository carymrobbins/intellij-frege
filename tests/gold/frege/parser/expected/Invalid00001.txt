Frege file
  PsiElement(CONID)('Invalid')
  PsiElement(VARID)('wher')
  PsiElement(VARID)('cats')
  PsiElement(CHAR)('=')
  PsiElement(STRCONST)('"dogs"')